common/core/desktop/tests/unit/kmx/013 - deadkeys.kmn

```diff
@@ -1,14 +1,14 @@
 c Description: Tests a simple deadkey rule in Unicode
-c keys: [SHIFT K_6][K_A]
-c expected: \u00E2
-c context: 
+c keys: [SHIFT K_6][K_A][SHIFT K_6][K_SHIFT][K_A][SHIFT K_6][RALT K_B][K_A][SHIFT K_6][K_CAPS][K_A]
+c expected: pass.pass.pass.pass.
+c context:
```

Comment:
I'd kinda prefer this test for shift, ctrl and alt to be in a separate .kmn which also refers to issue #5591, because in the future there will be confusion as to why all this is here 😁

Reply:
done

Reply:
done